packages/Testing/PHPUnit/AbstractRectorTestCase.php

```diff
@@ -94,7 +94,12 @@ protected function tearDown(): void
      */
     protected function yieldFilesFromDirectory(string $directory, string $suffix = '*.php.inc'): Iterator
     {
-        return StaticFixtureFinder::yieldDirectoryExclusively($directory, $suffix);
+        $counter = 0;
+        foreach (StaticFixtureFinder::yieldDirectoryExclusivelyWithRelativePathname($directory, $suffix) as $name => $parameters) {
+            yield '#' . $counter . ' ' . $name => $parameters;
+
+            ++$counter;
+        }
```

Comment:
it seems not working:

```bash
vendor/bin/phpunit rules-tests/TypeDeclaration/Rector/ClassMethod/AddArrayReturnDocTypeRector/AddArrayReturnDocTypeRectorTest.php --filter=#1
PHPUnit 9.5.20 #StandWithUkraine

No tests executed!
```

Reply:
Okay that is really strange seems like some strange phpunit internal behaviour how `#` is handled in the filter.